pkg/codeconfig/codeconfig.go

```diff
@@ -196,6 +196,8 @@ func (c *codeConfig) SpecFromWorkerPool(pool pool.WorkerPool) (*SpecResult, erro
 			httpWorkers = append(httpWorkers, &HttpWorker{
 				Port: w.GetPort(),
 			})
+		case *worker.WebsocketWorker:
+			// TODO: Add websocket worker to spec
```

Comment:
Should this TODO still be here?

Reply:
Yes, this will be for when we implement the dashboard support for websockets